readthedocs/proxito/views/hosting.py

```diff
@@ -514,6 +512,25 @@ def _v1(self, project, version, build, filename, url, request):
             },
         }
 
+        # Show the subprojects filter on the parent project and subproject
+        # TODO: Remove these queries and try to find a way to get this data
+        # from the resolver, which has already done these queries.
+        if project.subprojects.exists():
```

Comment:
You need to put a conditional here as it was in the above chunk: `if version:`, otherwise this block could generate 500.

Reply:
Would be useful in the docstring to note when the version is passed. It's not clear to me when it wouldn't be included.